### ISSUE TITLE
 Different libopencv_core3. and opencv_core2 version

### ISSUE THREAD

**Issue by installthetech** [installthetech/slit_scan_panorama#1]
This version of this code is tested on opencv 3.1. Other versions of the code for opencv2.x will be updated soon.

**Comment by zweipunktnull** [comment]
Hey ! 
Unfortunately still not working. Its now compilling, giving no errors, but no jpg is generated.
<img width="571" alt="code" src="https://user-images.githubusercontent.com/14869603/27047032-ec330b46-4fa6-11e7-9471-ee718e3f5f77.png">

**Comment by installthetech** [comment]
Try the latest updated repository. The output panorama will be generated in "output" directory.

**Comment by zweipunktnull** [comment]
Thanks for your answer. In this file is just the info.txt file.

Greetings
Chris ;)

**Comment by installthetech** [comment]
Try making the panorama on the newly cloned repo. Try it on the example video i.e. "data/pano_raw.mp4" and see if any output is generated?

**Comment by zweipunktnull** [comment]
really don't know what I am doing wrong.. here you can see what I am doing..

<img width="605" alt="out" src="https://user-images.githubusercontent.com/14869603/27048987-1aeec5e0-4fae-11e7-8da5-e600714ffa23.png">

**Comment by installthetech** [comment]
Use the command:
./bin/panorama data/pano_raw.mp4 pano.jpg

**Comment by zweipunktnull** [comment]
Still same result =( Im sorry when I disturb..

<img width="590" alt="out2" src="https://user-images.githubusercontent.com/14869603/27049273-1aebf684-4faf-11e7-9c48-0471126f577a.png">

**Comment by installthetech** [comment]
Which version of opencv are you using ?

**Comment by installthetech** [comment]
Did you clone the latest repo?

**Comment by zweipunktnull** [comment]
Im using last repo and opencv-3.2.0 ;)

**Comment by installthetech** [comment]
Did any window open when you ran the command? Ideally, it must open a window where the video must be played and and finally a window with the output path should appear

**Comment by zweipunktnull** [comment]
Unfortunately nothing is opening

**Comment by installthetech** [comment]
Can you rebuild your opencv with these cmake options:
cmake -D CMAKE_BUILD_TYPE=RELEASE -D CMAKE_INSTALL_PREFIX=/usr/local -D WITH_TBB=ON -D BUILD_NEW_PYTHON_SUPPORT=ON -D WITH_V4L=ON -D INSTALL_C_EXAMPLES=ON -D INSTALL_PYTHON_EXAMPLES=ON -D BUILD_EXAMPLES=ON -D WITH_OPENGL=ON ..

**Comment by installthetech** [comment]
Tell me whats the output of these two commands:
pkg-config --cflags opencv
pkg-config --libs opencv

**Comment by zweipunktnull** [comment]
Hey! Thank you so much for your answer. I installed KDE NEON fresh with that options and everything is working. Thank you so very much! Last question.. is a planar stitching also possible?

**Comment by zweipunktnull** [comment]
Is this stitching done with orb stitching algorithm?

**Comment by installthetech** [comment]
Firstly, yes, orb stitching is used in this algorithm. Secondly, the current repo supports panorama generation on videos (like mentioned in example) and live camera feed only. 
I would be glad if you could send some of your best panorama results here too.

**Comment by zweipunktnull** [comment]
Thank you for all your help! I will definitely do!

**Comment by zweipunktnull** [comment]
When I get this Error: 

QSettings::value: Empty key passed
QSettings::value: Empty key passed
Unable to get enough information to make panorama!
Saving the current progress in panorama.


Is the meaning that I did video too fast?

**Comment by zweipunktnull** [comment]
And with planar stitching I mean if its also working when I place my android phone on the window and drive and this could stitch a panorama while driving?, without moving the camera spherical?

**Comment by installthetech** [comment]
Yes, due to orb features, the camera needs to be moved slowly. Otherwise you will get this error due to unavailability of features. And, yes the panorama should be made if you drive "slowly" by keeping the phone on the window. However there's an upper limit to the size of panorama

**Comment by zweipunktnull** [comment]
Thank you so much for your answer, it helped me a lot. Is it also possible to change direction from right to left panorama stitching in the code?

**Comment by installthetech** [comment]
You need to get into the code and redraw the canvas using PRIMEPANO and DRAWIMAGE, which might take some amount of coding

**Comment by zweipunktnull** [comment]
Okay, I will give it a try ;) Thanks a lot

**Comment by installthetech** [comment]
Okay sure, Let me know if any other help is needed, and subscribe on installthetech for updates on upcoming posts

**Comment by zweipunktnull** [comment]
Thanks for your help.

I was trying now to apply the stitching algorithmus to a 384x288 pixel thermal camera. There is a problem because of this "low quality" its difficult to make a panorama. Is there a possibility in that source code to make the detection "lower", to combine also panorama if not every neighbour pixel is detected?

Thanks for your answer

**Comment by installthetech** [comment]
I have a created a new thread for this question
